rst_files/linear_algebra.rst

```diff
@@ -81,21 +76,20 @@ the point
 
 The following figure represents three vectors in this manner
 
+Activate the project environment, ensuring that ``Project.toml`` and ``Manifest.toml`` are in the same location as your notebook
+
 .. code-block:: julia
-    :class: test
 
-    using Test # Should put this near the top of every notebook.
+    using Pkg; Pkg.activate(@__DIR__); #activate environment in the notebook's location
```

Comment:
We can't have the activate stuff inside of the test block, otherwise it won't run when we generate the notebooks without tests.

But perhaps we need to have the activate in its own block before the `using Test` block?

Reply:
That's not a test block. The activate cell was after code so I moved it to the top before any code.

Reply:
Sorry, looks like I misread the diff